src/components/elements/side-tab.tsx

```diff
@@ -209,7 +208,7 @@ const SideTab = ({
                             {tab_header}
                         </Header>
                     )}
-                    <Tabs />
+                    {Tabs()}
```

Comment:
@niloo-fs Why have you changed this?

Reply:
They defined this function inside the component and it is not a jsx component so we had an error

Reply:
@niloo-fs The error is pretty clear about the issue 😅 React components can only return a single JSX element but this component is returning a list of elements, Just wrap a fragment around it and that’s it.

Also since this component is declared inside of another component it will be created again in each rerender, Please wrap it with `React.memo` to avoid this 🙏🏻

Reply:
HaHa you are right @farzin-deriv , will change this :-D